src/main/java/org/spongepowered/api/world/extent/ImmutableBiomeArea.java

```diff
@@ -24,11 +24,50 @@
  */
 package org.spongepowered.api.world.extent;
 
+import com.flowpowered.math.matrix.Matrix3d;
+import com.flowpowered.math.vector.Vector2i;
+import org.spongepowered.api.util.PositionOutOfBoundsException;
+
 /**
  * An area containing biomes that can be accessed but not modified.
+ * The data will never change.
  *
  * @see BiomeArea
  */
 public interface ImmutableBiomeArea extends BiomeArea {
 
+    /**
+     * Returns a new area that is the same or smaller than the current area.
+     * This does not copy the biomes, it only provides a new view of the
+     * storage.
+     *
+     * @param newMin The new minimum coordinates in this area
+     * @param newMax The new maximum coordinates in this area
+     * @return The new area with the new bounds
+     * @throws PositionOutOfBoundsException If the new minimum and maximum
+     *     are outside the current area
+     */
+    @Override ImmutableBiomeArea getBiomeView(Vector2i newMin, Vector2i newMax);
```

Comment:
The formatter was placing them on the same line... Maybe we should look into fixing it.



> On Jul 13, 2015, at 9:42 PM, Chris Sanders <notifications@github.com> wrote:
> 
> In src/main/java/org/spongepowered/api/world/extent/ImmutableBiomeArea.java:
> 
> >   *
> >   * @see BiomeArea
> >   */
> >  public interface ImmutableBiomeArea extends BiomeArea {
> >  
> > +    /**
> > +     * Returns a new area that is the same or smaller than the current area.
> > +     * This does not copy the biomes, it only provides a new view of the
> > +     * storage.
> > +     *
> > +     * @param newMin The new minimum coordinates in this area
> > +     * @param newMax The new maximum coordinates in this area
> > +     * @return The new area with the new bounds
> > +     * @throws PositionOutOfBoundsException If the new minimum and maximum
> > +     *     are outside the current area
> > +     */
> > +    @Override ImmutableBiomeArea getBiomeView(Vector2i newMin, Vector2i newMax);
> Styling issue. Override shouldn't be on same line.
> 
> —
> Reply to this email directly or view it on GitHub.
>